authentication/models/users.py

```diff
@@ -0,0 +1,120 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from historical_surveillance.models import School, District
+from django.contrib.auth.models import User, Group
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+
+class SchoolAdministrator(User):
+    school = models.ForeignKey(School, on_delete=models.CASCADE, null=True)
```

Comment:
A school adminisitrator is someone that will be allowed to login to this application to that's why they extend User. 

The School Admin will only be allowed to see data for one school - so for that reason they have a school class that needs to be set.

Reply:
looks good.